src/shared/telemetry/telemetryService.ts

```diff
@@ -21,14 +21,15 @@ import { ACCOUNT_METADATA_KEY, AccountStatus, COMPUTE_REGION_KEY } from './telem
 import { TelemetryLogger } from './telemetryLogger'
 import globals from '../extensionGlobals'
 import { ClassToInterfaceType } from '../utilities/tsUtils'
+import { shared } from '../utilities/functionUtils'
 
 export type TelemetryService = ClassToInterfaceType<DefaultTelemetryService>
 
 export class DefaultTelemetryService {
     public static readonly TELEMETRY_COGNITO_ID_KEY = 'telemetryId'
     public static readonly TELEMETRY_CLIENT_ID_KEY = 'telemetryClientId'
 
-    private static readonly DEFAULT_FLUSH_PERIOD_MILLIS = 1000 * 60 * 5 // 5 minutes in milliseconds
+    private static readonly DEFAULT_FLUSH_PERIOD_MILLIS = 1000 * 5 // 5 minutes in milliseconds
```

Comment:
This was changed from 5 minutes to 5 seconds. Was that intentional?

Reply:
no. I decreased the flush period to test what happens if I refresh a lot. does it get stuck.